Do not include /usr/include host paths

These were added upstream apparently for freebsd support, but trigger
compiler warnings during the build and could be dangerous, so remove
them.

Upstream-Status: Inappropriate [cross]

Signed-off-by: Paul Eggleton <paul.eggleton@linux.intel.com>

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,12 +1,5 @@
 cmake_minimum_required(VERSION 2.6.0)
 
-#freebsd support
-include_directories(SYSTEM /usr/local/include)
-link_directories(/usr/local/lib)
-link_directories(/usr/local/lib32)
-link_directories(/usr/local/lib64)
-SET(CMAKE_REQUIRED_INCLUDES ${CMAKE_REQUIRED_INCLUDES} /usr/local/include)
-
 #extract version from qmmp.h
 FILE(STRINGS "${CMAKE_CURRENT_SOURCE_DIR}/src/qmmp/qmmp.h"
      QMMP_VERSION_DATA REGEX "^#define[ \t]+QMMP_VERSION_[A-Z]+[ \t]+[0-9]+.*$")
